inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -85,9 +96,17 @@ ModelHeader GNAModelSerial::ReadHeader(std::istream &is) {
     if (header.version.major != HEADER_MAJOR) {
         THROW_GNA_EXCEPTION << "Imported file unsupported: major version should be == " << HEADER_MAJOR;
     }
+    if (header.version.minor != HEADER_MINOR) {
+        THROW_GNA_EXCEPTION << "Imported file unsupported: minor version should be == " << HEADER_MINOR;
```

Comment:
suggest better wording like unsupported minor version: << header.version.minor<<  ", but should be equal" to << HEADER_MAJOR;